devops/openshift/api-deploy.yml

```diff
@@ -252,7 +252,7 @@ objects:
             name: memory
             target:
               type: AverageValue
-              averageValue: "230M"
+              averageValue: "350M"
```

Comment:
Doesnt the average value of the HPA should be 9% of the Memory_Limit ? else it will create replicas when the memory request starts a pod from 384M. Or can you please explain, if my understanding is wrong. This applies to the setting below for queues and workers too.

Reply:
I am not sure if I am following but the PODs will not "request" the memory up to the "requested" amount. The request is what Openshift will secure to the POD, not necessarily what it will take right from the start. Please let me know if further clarification is needed.

Reply:
Thanks for the explanation in the call.